bin/fake_ifdh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python3 -I
```

Comment:
Sorry - I'm a bit confused.  Why do we need bin/fake_ifdh if we've got the fake_ifdh.py in lib?

Reply:
So the branch will merge.   If I remove it it tells me there are conflicts to resolve.   Once we merge this, we can dink the one in bin as another pull.

Reply:
Ah fun.  Okay - I can open an issue for that.